packages/schema/scripts/generate-schemas.mjs

```
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+
+/**
+ * Build-time script to generate JSON Schema files from Zod schemas
+ */
+
+import { mkdir, writeFile } from 'fs/promises'
+import { join, dirname } from 'path'
+import { fileURLToPath } from 'url'
+
+// Import the generation utilities
+import { generateAllSchemas, getJsonSchemaString } from '../dist/esm/index.js'
```

ESLint

[error] 'getJsonSchemaString' is defined but never used.

(no-unused-vars)